themes/digital.gov/layouts/shortcodes/do-dont-table.html

```diff
@@ -0,0 +1,51 @@
+{{/* 
+
+When setting up a **do-dont-table**, you need to make sure you have the correct formatting.
+
+1. Start with a a **do-dont-table** then nest **do-dont-row**
+2. It is best practice to provide a caption which describes the table contents, should be provided.
+3. {{ do-row }} & {{ dont-row }} short codes need to be paired or else the row will not render
+4. Place text, markdown and shortcodes between the {{ do-row }} & {{ dont-row }} short codes
+5. Make sure to close them with {{< /do-row >}} & {{< /dont-row >}}
+
+{{< do-dont-table caption="This is a table" >}}
+  {{< do-row >}} [Multilingual](https://digital.gov/communities/multilingual/) {{< /do-row >}}
```

Comment:
Good point, this was @ToniBonittoGSA markup originally.

We should be using the `link` short code for internal pages: `[DigitalGov User Experience Program]({{< link "digitalgov-user-experience-resources.md" >}}`

Reply:
In this case with nested short codes, the `link` markdown is required to make it play nicely with the formatter.
Using `link` or `ref` will just print the string instead of a hyper link.
It's an odd quirk of the markdown formatter that is a known issue.

Reply:
@nick-mon1 let's document that quirk if it's not already.

Reply:
I've updated the documentation to clarify this detail.

Reply:
thanks